aten/src/ATen/FunctionalInverses.cpp

```diff
@@ -228,6 +228,14 @@ Tensor FunctionalInverses::transpose_copy_int_inverse(const Tensor& base, const
     }
 }
 
+Tensor FunctionalInverses::_unsafe_view_copy_inverse(const at::Tensor & base, const at::Tensor & mutated_view, bool reapply_views, at::IntArrayRef size) {
```

Comment:
The size argument is ignored?

Reply:
yep - that's actually true for a bunch of other inverse functions too (like `view_copy_inverse()`). The invariant is that `mutated_view` is already of the same size as the `size` argument, and we want to go in the reverse direction, and make it the same size as the base.